packages/model/src/analytics.ts

```diff
@@ -76,10 +76,25 @@ export const timeRanges = [
   'all',
 ] as const;
 
+export const documentCategories = [
+  'all',
+  'article',
+  'bioinformatics',
+  'dataset',
+  'lab-resource',
+  'protocol',
+] as const;
+
 export type TimeRangeOption = (typeof timeRanges)[number];
+export type DocumentCategoryOption = (typeof documentCategories)[number];
+
+export type FilterAnalyticsOptions = {
+  timeRange?: TimeRangeOption;
+  documentCategory?: DocumentCategoryOption;
+};
 
 export type FetchAnalyticsOptions = FetchPaginationOptions & {
```

Comment:
this are generally used in various places and in many of those the `documentCategory` filter does not apply. Leave it though as I can address it in my PR

Reply:
Why would this be an issue since it can be undefined?